Added note about background platform channels not being available for iOS yet (#6751)
flutter/website: src/development/platform-integration/platform-channels.md

destined for the platform side, they need to be invoked on the root Isolate. The
platform side's handlers can execute on the platform's main thread or they can
execute on a background thread if a Task Queue is used. The result of the
platform side handlers can be invoked asynchronously and on any thread when the

> Task Queue API is available, otherwise they must be invoked on the platform

Reviewer: ```suggestion
Task Queue API is available; otherwise, they must be invoked on the platform
```

thread.

> {{site.alert.note}}The Task Queue API is available for Android on the `stable`
> channel. For iOS, it is currently available on the `master` channel.

Reviewer: ```suggestion
{{site.alert.note}}
  The Task Queue API is available for Android on the `stable`
  channel. For iOS, it is currently available on the `master` channel.
{{site.alert.end}}
```

Author: Did they change the requirement for the end tag at some point?  I swore these were rendering correctly previously.

> channel. For iOS, it is currently available on the `master` channel.

Reviewer: In a couple places in this PR the text says "currently". But, can you be more specific? "As of release xxx".

Author: done

> {{site.alert.note}}On Android, the platform's main thread is sometimes called

Reviewer: Please make sure that this merges correctly, as it didn't want to let me modify the follow on sentences that were part of the note.

```suggestion
{{site.alert.note}}
  On Android, the platform's main thread is sometimes called
  the "main thread", but it is technically defined as [the UI thread][].
  Annotate methods that need to be run on the UI thread with `@UiThread`.
  On iOS, this thread is officially referred to as [the main thread][].
{{site.alert.end}}
```

> the "main thread", but it is technically defined as [the UI thread][]. Annotate

Reviewer: ```suggestion
```

Author: done

```

In Swift:

> {{site.alert.note}}The Task Queue API is only currently available on the
> `master` channel for iOS .

Reviewer: ```suggestion
{{site.alert.note}}
  In release, xxx, the Task Queue API for iOS is only available on the
 `master` channel.
 {{site.alert.end}}
```

In Objective-C:

> {{site.alert.note}}The Task Queue API is only currently available on the
> `master` channel for iOS .

Reviewer: ```suggestion
{{site.alert.note}}
  As of release xxx, the Task Queue API for iOS is only available on the
`master` channel.
{{site.alert.end}}
```

Author: done
